### ISSUE TITLE
[danfossairunit] error during startup after OH3 migration

### ISSUE THREAD

**Issue by pravussum** [openhab/openhab-addons#9568]
## Expected Behavior
The binding starts up without errors. 
## Current Behavior
During startup with OH version 3 the below warning is reported in the logs. No airuinit connection is established. 

```
2020-12-28 22:45:20.545 [WARN ] [ig.xml.osgi.XmlDocumentBundleTracker] - The XML document '/OH-INF/thing/thing-types.xml' in module 'org.openhab.binding.danfossairunit' could not be parsed: The attribute 'advanced' of the node 'channel-group-type' is not supported or exists multiple times!
---- Debugging information ----
message             : The attribute 'advanced' of the node 'channel-group-type' is not supported or exists multiple times!
class               : org.openhab.core.thing.xml.internal.ChannelGroupTypeXmlResult
required-type       : org.openhab.core.thing.xml.internal.ChannelGroupTypeXmlResult
converter-type      : org.openhab.core.thing.xml.internal.ChannelGroupTypeConverter
path                : /thing-descriptions/channel-group-type[4]
line number         : 91
class[1]            : java.util.ArrayList
required-type[1]    : java.util.ArrayList
converter-type[1]   : com.thoughtworks.xstream.converters.collections.CollectionConverter
class[2]            : org.openhab.core.thing.xml.internal.ThingDescriptionList
required-type[2]    : org.openhab.core.thing.xml.internal.ThingDescriptionList
converter-type[2]   : org.openhab.core.thing.xml.internal.ThingDescriptionConverter
version             : 1.4.13
-------------------------------
com.thoughtworks.xstream.converters.ConversionException: The attribute 'advanced' of the node 'channel-group-type' is not supported or exists multiple times!
---- Debugging information ----
message             : The attribute 'advanced' of the node 'channel-group-type' is not supported or exists multiple times!
class               : org.openhab.core.thing.xml.internal.ChannelGroupTypeXmlResult
required-type       : org.openhab.core.thing.xml.internal.ChannelGroupTypeXmlResult
converter-type      : org.openhab.core.thing.xml.internal.ChannelGroupTypeConverter
path                : /thing-descriptions/channel-group-type[4]
line number         : 91
class[1]            : java.util.ArrayList
required-type[1]    : java.util.ArrayList
converter-type[1]   : com.thoughtworks.xstream.converters.collections.CollectionConverter
class[2]            : org.openhab.core.thing.xml.internal.ThingDescriptionList
required-type[2]    : org.openhab.core.thing.xml.internal.ThingDescriptionList
converter-type[2]   : org.openhab.core.thing.xml.internal.ThingDescriptionConverter
version             : 1.4.13
-------------------------------
	at org.openhab.core.config.xml.util.ConverterAttributeMapValidator.readValidatedAttributes(ConverterAttributeMapValidator.java:117) ~[?:?]
	at org.openhab.core.config.xml.util.ConverterAttributeMapValidator.readValidatedAttributes(ConverterAttributeMapValidator.java:82) ~[?:?]
	at org.openhab.core.thing.xml.internal.AbstractDescriptionTypeConverter.unmarshal(AbstractDescriptionTypeConverter.java:184) ~[?:?]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readBareItem(AbstractCollectionConverter.java:132) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(AbstractCollectionConverter.java:117) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(CollectionConverter.java:98) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:91) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(CollectionConverter.java:85) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(CollectionConverter.java:80) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50) ~[bundleFile:1.4.13]
	at org.openhab.core.thing.xml.internal.ThingDescriptionConverter.unmarshal(ThingDescriptionConverter.java:53) ~[?:?]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convert(TreeUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(AbstractReferenceUnmarshaller.java:72) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:66) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(TreeUnmarshaller.java:50) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.TreeUnmarshaller.start(TreeUnmarshaller.java:134) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.core.AbstractTreeMarshallingStrategy.unmarshal(AbstractTreeMarshallingStrategy.java:32) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1404) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.XStream.unmarshal(XStream.java:1383) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:1334) ~[bundleFile:1.4.13]
	at com.thoughtworks.xstream.XStream.fromXML(XStream.java:1288) ~[bundleFile:1.4.13]
	at org.openhab.core.config.xml.util.XmlDocumentReader.readFromXML(XmlDocumentReader.java:106) ~[bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker.parseDocuments(XmlDocumentBundleTracker.java:396) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker.processBundle(XmlDocumentBundleTracker.java:382) [bundleFile:?]
	at org.openhab.core.config.xml.osgi.XmlDocumentBundleTracker$2.run(XmlDocumentBundleTracker.java:347) [bundleFile:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by jlaur** [comment]
@pravussum - this issue can be closed with #9569, right?

**Comment by pravussum** [comment]
Exactly.
